### PR TITLE
Run pull on init

### PR DESCRIPTION
**Pull request by SferaDev**
Fixes https://github.com/xataio/client-ts/issues/992

### CODE REVIEW

**Comment by kostasb** [comment]
While init-ing on a fresh directory, there's an error due to the migrations dir not being there

> no such file or directory, stat '/Users/kostas/testdir/.xata/migrations'


```
xataCli init
Need to install the following packages:
  @xata.io/cli@0.0.0-alpha.v38ffdc3
Ok to proceed? (y) y
🦋 Initializing project... We will ask you some questions.

✔ Select a workspace › ws
✔ Select a database or create a new one › test
✔ Create .gitignore and ignore .env? … yes
✔ Generate code and types from your Xata database › TypeScript
✔ Choose the output path for the generated code … src/xata.ts

Setting up Xata...

Created Xata config: .xatarc

Creating .env file
  set XATA_API_KEY=xau_*********************************
  set XATA_BRANCH=main

Added .env file to .gitignore

    Error: ENOENT: no such file or directory, stat '/Users/kostas/testdir/.xata/migrations'
    Code: ENOENT

```

**Comment by kostasb** [comment]
Didn't step on any other issues with this approach, pull on init works well (as long as the .xata/migrations directory exists)

**Comment by SferaDev** [comment]
Okay will fix the problem with directory

**Comment by SferaDev** [comment]
@kostasb Can you try again when the new build finishes?

**Comment by kostasb** [comment]
> @kostasb Can you try again when the new build finishes?

Worked well this time, thanks!

File: cli/src/commands/init/index.ts [hunk 1]
```diff
@@ -158,6 +159,8 @@ export default class Init extends BaseCommand<typeof Init> {
       await this.deploySchema(workspace, region, database, branch, schema);
     }
 
+    // Run pull to retrieve remote migrations, remove any local migrations, and generate code
```
Comment (richardgill) [inline cli/src/commands/init/index.ts:162]: This comment made me think pull generates code? 

If it does, should we remove the codegen line?

**Review by richardgill — APPROVED** [review]
Left a comment.

LGTM

File: cli/src/commands/init/index.ts (hunk 1 above)
Comment (SferaDev) [reply cli/src/commands/init/index.ts:162]: It does but the config is not saved yet, so we need to call them both.

**Pull request merged**